LBHackney-IT/lbh-fss-portal-frontend
# Feature/add new service

**description**

add functionality to allow a user to add a new service 

**test instruction**

- pull this branch 
- `yarn install`
- navigate to **/services/add**
- navigate through each page of the form and check functionality works and is consistent with designs 
- check hidden fields appear as appropriate (as shown on figma designs) 
- pay particular attention to validation logic on page 2 (service locations page) i.e. you must enter a location, you can enter multiple locations, locations cant be duplicate, you can add a location and change postcode and/or remove location as many times as you like without breaking the validation or getting any console errors 
- you can add and remove an image on page 5 (service image page)

figma designs: 

https://www.figma.com/file/t6fanDKvLkCHJsY85ivFTF/Services-Directory?node-id=586%3A8553

![image](https://user-images.githubusercontent.com/69238887/94475639-47397680-01c7-11eb-97f1-074ba5120f3f.png)


**ticket**

https://app.clickup.com/t/9e5u7e
https://app.clickup.com/t/9e5u7j
https://app.clickup.com/t/9e5u7q
https://app.clickup.com/t/78ntbf
https://app.clickup.com/t/78ntch
https://app.clickup.com/t/9e5u7g
https://app.clickup.com/t/9e5u7n
https://app.clickup.com/t/9e5u7c
https://app.clickup.com/t/9e5u7x
https://app.clickup.com/t/9e5u7u



**note**
if you skip back to page 2 using the menu navigation the postcode data will not be displayed - this will be picked up on a subsequent commit
